sklearn/feature_selection/from_model.py

```diff
@@ -169,6 +169,12 @@ def fit(self, X, y=None, **fit_params):
         self.estimator_.fit(X, y, **fit_params)
         return self
 
+    @property
+    def threshold_(self):
+        scores = _get_feature_importances(self.estimator_, self.norm_order)
```

Comment:
we explicitly allow people to change a threshold after fitting the estimator. There's a test for that.